.github/workflows/centraldb_angular_frontend_test.yaml

```diff
@@ -42,3 +42,63 @@ jobs:
           cd components/centraldashboard-angular/frontend/
           npm i
           npm run test:prod
+
+  run-tests-in-chrome:
+    name: UI tests in chrome
+    runs-on: ubuntu-20.04
```

Comment:
Should we use `runs-on: ubuntu-latest`  GitHub-hosted runner similar to the other jobs ([here](https://github.com/arrikto/kubeflow/blob/84afe05474d5fea0ff4adac275fc4ec3e7013e3c/.github/workflows/centraldb_angular_frontend_test.yaml#L10) and [here](https://github.com/arrikto/kubeflow/blob/84afe05474d5fea0ff4adac275fc4ec3e7013e3c/.github/workflows/centraldb_angular_frontend_test.yaml#L34))?

Reply:
This is mostly because we wanted to make sure that the base would always contain both Firefox and Chrome browsers. Although indeed now even `ubuntu-22.04` has the Firefox installation we wanted to make sure the tests wouldn't break from a change in ubuntu latest.

And we'll be periodically updating the bases of these actions
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#browsers-and-drivers
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#browsers-and-drivers